decentraland/unity-explorer
# Fix: colliders visibility switch

## What does this PR change?
fix #1190 
Add colliders enabling switch based on visibility.

## How to test the changes?

1. Check the original bug report for testing. Basically you shouldn't be able to click on hidden plates of the carousel.
2. 
## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
